src/main/java/com/aws/greengrass/logmanager/LogManagerService.java

```diff
@@ -144,7 +144,17 @@
     private synchronized void processConfiguration(Map<String, Object> configTopicsPojo) {
         Map<String, ComponentLogConfiguration> newComponentLogConfigurations = new ConcurrentHashMap<>();
         configTopicsPojo.computeIfPresent(COMPONENT_LOGS_CONFIG_TOPIC_NAME, (s, o) -> {
-            if (o instanceof ArrayList) {
+            if (o instanceof Map) {
```

Comment:
Just 1 small change to request.

Instead of switching on this single topic, let's create a new configuration topic for the map type specifically. `componentLogsConfigurationMap`

so 
```
configTopicsPojo.computeIfPresent("componentLogsConfigurationMap", (s, o) -> {
  if (o instanceof Map) {
    .... same logic you added ....
 }
});
```

Reply:
+1 on this - the same approach (new topic) was used for shadow: https://github.com/aws-greengrass/aws-greengrass-shadow-manager/pull/93